common/src/java/com/zimbra/common/util/Log.java

```diff
@@ -134,15 +136,15 @@ public boolean isInfoEnabled() {
     }
 
     public boolean isWarnEnabled() {
-        return getLogger().isEnabledFor(org.apache.log4j.Level.WARN);
+        return getLogger().isEnabled(org.apache.logging.log4j.Level.WARN);
```

Comment:
Can't we set it as Level.WARN instead of org.apache.logging.log4j.Level.WARN

Reply:
In this class, an enum Level is defined that is the reason fully qualified name is used.